React/Fabric/Mounting/ComponentViews/Text/RCTParagraphComponentView.mm

```diff
@@ -19,7 +20,7 @@
 using namespace facebook::react;
 
 @implementation RCTParagraphComponentView {
-  SharedParagraphLocalData _paragraphLocalData;
+  SharedParagraphStateData _paragraphStateData;
```

Comment:
You have to store the whole State, not only `data` part (not sure exactly why, it just feels better).

You can access this type as `ParagraphShadowNode::ConcreteState::Shared`.

Reply:
Oh, that's why you needed that `SharedParagraphStateData ` type. So, you don't need it. ;)

The point of State object is that you can initiate state update from that. We don't need that for Text, but we will need that for TextInput.